.gitignore

```diff
@@ -6,5 +6,6 @@ build/
 dist/
 .vscode
 venv
+.venv
```

Comment:
unrelated to the fix, but I added this because it's common to have the virtual environment folder as hidden